index.js

```diff
@@ -1,11 +1,44 @@
 'use strict';
 const fs = require('fs');
+const os = require('os');
 const path = require('path');
 const childProcess = require('child_process');
 const runApplescript = require('run-applescript');
 const pify = require('pify');
 const rimraf = require('rimraf');
-const xdgTrashdir = require('xdg-trashdir');
```

Comment:
I have access to that repo, so you could just do a pull request there and I'll review.